src/app/project/tests/spec/SpecFeatureDetails.js

```diff
@@ -43,24 +49,111 @@ require([
             });
         });
         describe('onFeatureSelected', function () {
-            it('places template in contents', function () {
-                widget.onFeatureSelected({});
+            beforeEach(function () {
+                if (widget) {
+                    destroy(widget);
+                }
+            });
+
+            it('places template in contents', function (done) {
+                stubModule('app/project/FeatureDetails', {
```

Comment:
The `stubModule` stuff could be refactored into the `beforeEach` method for all three of these tests. This would eliminate a bunch of duplicate code.